ark-plonk-book/src/chapter_1.md

```diff
@@ -0,0 +1,373 @@
+# Chapter 1: Background Material
+
+## Groups
+A binary operation $*$ on a set $G$ is a mapping from $G\times G$ to $G$, which associates to elements $x$ and $y$ of $G$ a third element $x*y$ of $G$. 
+
+#### Definition: 
+A group $(G, *)$ consists of a set $G$ together with a binary operation $*$ for which the following properties are satisfied: 
+
+* Associativity: $(x*y)*z=x*(y*z),\; \forall x,y,z\in G $ 
+* Neutral element: $\exists! \; e \in G,\; e*x=x=x*e,\; \forall x\in G$
+* Inverse element: $\forall x\in G, \; \exists! \; x'\in G,\; x*x'=e=x'*x$ where $e$ is the neutral element of $G$. 
+
+A group $G$ is Abelian (or commutative) if: $x*y=y*x,\forall x,y\in G$
+
+### Cyclic groups:
+ 
+#### Definition: 
+A group $G$ is said to be cyclic, with generator $x$, if every element of $G$ is of
+the form $x^n$ for some integer $n$.
+
+## Fields
+ 
+A field $F$ is a set with two binary operations $+$ and $*$ that satisfies the following field axioms:
+ 
+* Closure under addition:  $\forall x,y\in F,\; x+y\in F$
+* Closure under multiplication:  $\forall x,y\in F,\; x*y\in F$
+* $(F, +)$ is a commutative and associative group with identity element $0$
+* $(F-\{0\},* )$ is a commutative and associative group with identity element $1$
+* Additive inverses:  $\forall x\in F,  y\in F$ such that  $x+y=y+x=0$
+* Multiplication inverses:  $\forall x\in F$ such that $x\neq 0, \; \exists \; y\in F$ such that $x*y=y*x=1$, 
```

Comment:
Since groups are always associative, we don't need the extra qualifier here. We can also remove the commutativity constraints in lines 29 and 30, and just have `x+y=0` and `x*y=1`.